arcjet/index.ts

```diff
@@ -21,13 +21,15 @@ import {
   ArcjetSlidingWindowRateLimitRule,
   ArcjetShieldRule,
   ArcjetLogger,
+  ArcjetRateLimitRule,
 } from "@arcjet/protocol";
 import { ArcjetBotTypeToProtocol } from "@arcjet/protocol/convert.js";
 import { Client } from "@arcjet/protocol/client.js";
 import * as analyze from "@arcjet/analyze";
 import * as duration from "@arcjet/duration";
 import ArcjetHeaders from "@arcjet/headers";
 import { runtime } from "@arcjet/runtime";
+import { RateLimitRule } from "@arcjet/protocol/gen/es/decide/v1alpha1/decide_pb";
```

Comment:
We don't want to import anything from `gen/es`. Instead we want to abstract over it. We're planning to make larger changes to the network layer and I don't want to leak the implementation into our SDKs

Reply:
It actually looks like this might just be an auto import that didn't get removed.
```suggestion
```